src/services/document-service.js

```diff
@@ -82,16 +84,20 @@ class DocumentService {
     this.userService = userService;
   }
 
-  getAllDocumentsMetadata() {
-    return this.documentStore.find({}, { sort: lastUpdatedFirst, projection: metadataProjection });
-  }
-
-  getDocumentsMetadataByKeys(documentKeys) {
-    return this.documentStore.find({ _id: { $in: documentKeys } }, { sort: lastUpdatedFirst, projection: metadataProjection });
+  getAllDocumentsMetadata({ includeArchived } = {}) {
```

Comment:
This could be more generic:
 (extraFilters = {})

Then you have 
const filter = {...extraFilters}

Reply:
Not really, `includeArchived` is not a db filter as it's not part of the model. `archived` is part of the model. I specifically did not want the controller to call this method passing in the DB filter, since it's out of its scope to handle the filtering specific to the DB type used.